File: ecopoints/app/src/main/java/at/htl/ecopoints/ProfileActivity.java
package at.htl.ecopoints;

import android.os.Bundle;
import android.util.Log;

import androidx.activity.ComponentActivity;

import javax.inject.Inject;

import at.htl.ecopoints.ui.layout.ProfileView;
import dagger.hilt.android.AndroidEntryPoint;

@AndroidEntryPoint
public class ProfileActivity extends ComponentActivity {

    private static final String TAG = ProfileActivity.class.getSimpleName();

    @Inject
    ProfileView profileView;

    @Override
    public void onCreate(Bundle savedInstanceState) {
        Log.i(TAG, "onCreate()");
        super.onCreate(savedInstanceState);

        profileView.compose(this);
    }
}

File: ecopoints/app/src/main/java/at/htl/ecopoints/activity/ui/theme/ProfileActivity.kt
package at.htl.ecopoints.activity.ui.theme

import android.os.Bundle
import android.view.LayoutInflater
import android.view.ViewGroup
import android.widget.ScrollView
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import androidx.compose.foundation.layout.Box
import androidx.compose.foundation.layout.fillMaxSize
import androidx.compose.material3.MaterialTheme
import androidx.compose.material3.Surface
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.runtime.mutableStateOf
import androidx.compose.runtime.remember
import androidx.compose.ui.Alignment
import androidx.compose.ui.Modifier
import androidx.compose.ui.platform.LocalContext
import androidx.compose.ui.tooling.preview.Preview
import at.htl.ecopoints.R
import at.htl.ecopoints.activity.ui.theme.ui.theme.EcoPointsTheme
import at.htl.ecopoints.navigation.BottomNavBar
import com.google.android.material.composethemeadapter.MdcTheme
import androidx.compose.ui.viewinterop.AndroidView

class ProfileActivity : ComponentActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_profile)

    }

}
